File: frontend/src/app/dto/attachment-ticket/attachment-ticket.data.dto.ts
export class AttachmentTicketDataDto{
    private _id! : number
	private _fileId! : string
	private _ticketId! : string
    private _ver! : number

    public get fileId(): string {
        return this._fileId;
    }

    public set fileId(fileId: string) {
        this._fileId = fileId;
    }

    public get ticketId(): string {
        return this._ticketId;
    }

    public set ticketId(ticketId: string) {
        this._ticketId = ticketId;
    }
    
    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }


    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }

}

File: backend/src/main/java/com/lawencon/ticketms/dto/attachmentticket/AttachmentTicketDataDto.java
package com.lawencon.ticketms.dto.attachmentticket;

public class AttachmentTicketDataDto {
	private Long id;
	private Long fileId;
	private Long ticketId;
	private Integer ver;
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public Long getFileId() {
		return fileId;
	}
	public void setFileId(Long fileId) {
		this.fileId = fileId;
	}
	public Long getTicketId() {
		return ticketId;
	}
	public void setTicketId(Long ticketId) {
		this.ticketId = ticketId;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
